ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -571,4 +571,14 @@ FAN_SS_AL:
   - control_mode
   - run_mode
   - schedule_run_command
-  - control_status
\ No newline at end of file
+  - control_status
+
+FAN_SSSCFASR:
+  description: "Basic Exhaust fan with start/stop, speed command, fail alarm, scheduled run"
+  uses:
+  - exhaust_fan_speed_percentage_command
+  implements:
+  - CONTROL
```

Comment:
remove; this goes with abstract types only.

Reply:
noted sir